mediathread/projects/templatetags/user_projects.py

```diff
@@ -101,3 +101,8 @@ def date_format_change(date):
     newFormat = ''.join(newDate.split())
 
     return re.sub(r'(?<=[,])(?=[^\s])', r' ', newFormat)
+
+
+@register.simple_tag
+def show_discussion_response(status, comments):
```

Comment:
It's not clear to me from the code what the `comments` list is. Is this a list of comments? Comment lengths? A note would be good to clarify this.

Reply:
Good point. I've updated to clarify.